lib/driver.js

```diff
@@ -686,21 +686,24 @@ class XCUITestDriver extends BaseDriver {
       isHeadless: !!this.opts.isHeadless,
     };
 
+    runOpts.devicePreferences = {
+      simulatorWindowCenter: this.opts.simulatorWindowCenter,
+    };
     // This is to workaround XCTest bug about changing Simulator
     // orientation is not synchronized to the actual window orientation
     const orientation = _.isString(this.opts.orientation) && this.opts.orientation.toUpperCase();
     switch (orientation) {
       case 'LANDSCAPE':
-        runOpts.devicePreferences = {
+        runOpts.devicePreferences = Object.assign({}, runOpts.devicePreferences, {
```

Comment:
I don't understand why this `Object.assign` is necessary. The `runOpts` object is only in scope in this function.

Reply:
yes, you are right. The thing might be simplified